src/Parser/Warrior/Arms/Modules/Talents/Rend.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import Wrapper from 'common/Wrapper';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Analyzer from 'Parser/Core/Analyzer';
+
+const EXECUTE_RANGE = 0.2;
+
+class RendAnalyzer extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    enemies: Enemies,
+  };
+
+  rendsInExecuteRange = 0;
+  targets = [];
+  failedTargets = [];
+
+  on_initialized() {
+		this.active = this.combatants.selected.hasTalent(SPELLS.REND_TALENT.id);
+	}
+
+  on_byPlayer_cast(event) {
+    if(SPELLS.REND_TALENT.id === event.ability.guid) {
+      // If Rend is used flag the target to check their health on the next damage event.
+      this.targets.push(event.targetID);
+    }
+  }
+
+  on_byPlayer_damage(event) {
+    const targetId = event.targetID;
+
+    if(SPELLS.REND_TALENT.id === event.ability.guid && this.targets.indexOf(targetId) !== -1) {
+      // Check if the target's health was within execute range.
+      if(event.hitPoints / event.maxHitPoints < EXECUTE_RANGE) {
+        if(this.failedTargets.indexOf(targetId) === -1) {
+          // If this is the first time Rend was used on this target in execute range we flag that target.
+          // This is to prevent false positives from Rend being applied just as the target enters execute range.
+          this.failedTargets.push(targetId);
+        } else {
+          // Increment the counter
+          this.rendsInExecuteRange += 1;
+        }
+      }
+
+      // Unflag the target for checking.
+      this.targets.splice(this.targets.indexOf(targetId), 1);
+    }
+  }
+
+  get executeRendsThresholds() {
+    return {
+			actual: this.rendsInExecuteRange,
```

Comment:
I'd suggest using per minute or percent of total rend casts for this to keep the threshold relative to encounter duration.

Reply:
Avoiding using rend during execute range isn't a mechanic that gets more difficult with encounter length. I think it's best as a hard suggestion so if the player makes the mistake even once they're guaranteed to receive feedback about it.

Reply:
What about encounters with adds or multiple bosses dying at different times? There's also the fact that messing up 1 rend is significantly more impact full on a 1 min. encounter than a 10 min. encounter.